guides/V1_UpdateMessage.md

```diff
@@ -9,7 +9,17 @@ is: 'experimental'
 ## Concepts
 
 The _updateMessage_ mutation allows the API caller to change the contents of a message.  The mutation accepts a message id (id) and the new content (content) as arguments, and makes the request on behalf of the caller, adding the specified new content to the message replacing the previous message content.
-Note: Only the author of a message can update its content.  Updating the content of messages not created by the calling user will receive a 403 Forbidden response.
+
+Notes:
+<br>
+ - Only the author of a message can update its content.  Updating the content of messages not created by the calling user will receive a **403 Forbidden** response.
+<br>
+ - Existing annotations will be stripped from an updated message.  Apps are expected to regenerate any annotation using the new content.  Mention annotations are automatically regenerated based on the new content.
```

Comment:
You can probably mention that these annotations will be sent as `message-annotation-added` events.